Sources/SectionScrubber.swift

```diff
@@ -224,7 +224,11 @@ public class SectionScrubber: UIView {
             return indexPath
         }
         for indexPath in collectionView.indexPathsForVisibleSupplementaryElementsOfKind(UICollectionElementKindSectionHeader) {
-            let view = collectionView.supplementaryViewForElementKind(UICollectionElementKindSectionHeader, atIndexPath: indexPath)
+            #if swift(>=2.3)
+                let view = collectionView.supplementaryViewForElementKind(UICollectionElementKindSectionHeader, atIndexPath: indexPath)!
```

Comment:
The Swift 2.2 code will be removed once Xcode 8 is released, I think it's better to not do much refactoring to have 2.3 support.